pkg/operator/resourcesynccontroller/core.go

```diff
@@ -18,7 +19,8 @@ func CombineCABundleConfigMaps(destinationConfigMap ResourceLocation, lister cor
 	certificates := []*x509.Certificate{}
 	for _, input := range inputConfigMaps {
 		inputConfigMap, err := lister.ConfigMaps(input.Namespace).Get(input.Name)
-		if apierrors.IsNotFound(err) {
+		if apierrors.IsNotFound(err) && !input.Required {
+			klog.V(2).Infof("Optional ConfigMap %s/%s doesn't exist yet. Skipping.", input.Namespace, input.Name)
```

Comment:
even higher than 2?

Reply:
What level would you like to see it on? I don't think this deserves that level of attention as the CM is optional so it's valid to be missing.